controllers/storagecluster/reconcile.go

```diff
@@ -635,47 +635,114 @@ func remove(slice []string, s string) (result []string) {
 	return
 }
 
-// ensureCreated ensures if the osd removal job template exists
-func (obj *ocsJobTemplates) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.StorageCluster) error {
-	osdCleanUpTemplate := &openshiftv1.Template{
+func osdCleanUpTemplate() *openshiftv1.Template {
+
+	NameSpace := "openshift-storage"
+	jobTemplateName := "ocs-osd-removal"
+	osdCleanupArgs := []string{
+		"ceph",
+		"osd",
+		"remove",
+		"--osd-ids=${FAILED_OSD_IDS}",
+	}
+	return &openshiftv1.Template{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      "ocs-osd-removal",
-			Namespace: sc.Namespace,
+			Namespace: "openshift-storage",
 		},
-	}
-	_, err := controllerutil.CreateOrUpdate(context.TODO(), r.Client, osdCleanUpTemplate, func() error {
-		osdCleanUpTemplate.Objects = []runtime.RawExtension{
-			{
-				Object: newCleanupJob(sc),
-			},
-		}
-		osdCleanUpTemplate.Parameters = []openshiftv1.Parameter{
+		Parameters: []openshiftv1.Parameter{
 			{
 				Name:        "FAILED_OSD_IDS",
 				DisplayName: "OSD IDs",
 				Required:    true,
 				Description: `
-The parameter OSD IDs needs a comma-separated list of numerical FAILED_OSD_IDs 
-when a single job removes multiple OSDs. 
-If the expected comma-separated format is not used, 
-or an ID cannot be converted to an int, 
-or if an OSD ID is not found, errors will be generated in the log and no OSDs would be removed.`,
+	The parameter OSD IDs needs a comma-separated list of numerical FAILED_OSD_IDs 
+	when a single job removes multiple OSDs. 
+	If the expected comma-separated format is not used, 
+	or an ID cannot be converted to an int, 
+	or if an OSD ID is not found, errors will be generated in the log and no OSDs would be removed.`,
```

Comment:
Any reason you are adding this whitespace? If you're going to do it at least indent it all to line up with the `Description` line like you've done below.